front/lib/api/assistant/actions/process.ts

```diff
@@ -247,18 +247,32 @@ export class ProcessConfigurationServerRunner extends BaseActionConfigurationSer
       DustProdActionRegistry["assistant-v2-process"].config
     );
 
+    // Transform each data source in actionConfiguration.dataSources by adding a
+    // `dataSourceOrViewId` field. This field will hold the value of `dataSourceViewId`
+    // if it exists; otherwise, it will fall back to `dataSourceId`
+    const actionConfigurationDataSources = actionConfiguration.dataSources.map(
+      (d) => {
+        const { dataSourceViewId, dataSourceId, ...rest } = d;
```

Comment:
nit: why not destruct in args directly ?

Reply:
🤷